packages/backend/src/machine-utils.spec.ts

```diff
@@ -51,6 +51,11 @@ vi.mock('@podman-desktop/api', async () => {
         };
       },
     },
+    env: {
+      isMac: true,
```

Comment:
it seems it'll test only macos case in the unit tests ?

Usually it's a vi.fn function and there are tests for each os

Reply:
There are no tests that are actually platform dependent (yet), but agree, that's better for now and more obvious later. Updated.